bindings/python/CHANGELOG.md

```diff
@@ -4,11 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.12.1] 
+
+- [#938] **Breaking change**. Reverted (https://github.com/huggingface/transformers/issues/16520)
+
 ## [0.12.0] 
 
 Bump minor version because of a breaking change.
 
-- [#938] **Breaking change**. Decoder trait is modified to be composable. This is only breaking if you are using decoders on their own. tokenizers should be error free.
+- [#938] ~~**Breaking change**. Decoder trait is modified to be composable. This is only breaking if you are using decoders on their own. tokenizers should be error free.~~
```

Comment:
Thought the same @SaulLu :)

Reply:
Well I though it might be nice to not be polluted by this bold statement that is clearly not present in the release anymore.

I tend to read changelogs by looking for bold text and ignore everything that doesn't directly concern me (usually when I look at changelogs I really focus on the thing that broke my build). Since the change is not present anymore, I would like to avoid confusion by making that explicit that the change is not include so not to be taken into account.

Maybe `[REVERTED] `  tag would be better than crossing it out ? Wdyt ?